OsmAnd/src/net/osmand/plus/myplaces/tracks/dialogs/SplitSegmentDialogFragment.java

```diff
@@ -244,6 +288,7 @@ public void trackSplittingFinished(boolean success) {
 					selectedGpxFile.setSplitGroups(groups, app);
 				}
 				updateContent();
+				app.getGpxDisplayHelper().removeSplitTrackTask(selectedGpxFile);
```

Comment:
Better move this removal to backend, UI could forget to use it, which will lead to new issues